src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,45 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+
 public class SalaryInfo {
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        StringBuilder stringBuilder = new StringBuilder(
+                "Report for period " + dateFrom + " - " + dateTo);
+        int space = 1;
+        LocalDate localDateFrom = LocalDate.of(
```

Comment:
Please fetch the whole date as a single substring and use DateTimeFormatter to parse it.

Reply:
not fixed